R/utils.R

```diff
@@ -199,9 +199,8 @@ enw_example <- function(type = c(
 #' )
 coerce_date <- function(dates) {
   if (is.null(dates)) {
-    return(data.table::as.IDate(.Date(numeric())))
+    return(data.table::as.IDate(c()))
```

Comment:
apparently, the linting gods are displeased. `integer()` should work fine instead of `c()` though apparently, so does `NULL` directly.

However, n.b. length(NULL) == 0 as well - that suggests the 201-203 can be eliminated entirely, provided that behavior of length isn't a new R version feature.

Reply:
(because 204-206 behavior provides the desired check => result)

Reply:
ruh roh and they are vicious gods

Reply:
> (because 204-206 behavior provides the desired check => result)

I'm a bit hesitant to depend on this on its own and also starting to feel very tired of this PR (;)). I'd suggest we leave as is and maybe address if it becomes a problem?